build_projects/shared-build-targets-utils/Utils/BuildVersion.cs

```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```

Comment:
It looks like there are 2 "dead" code places that use it:

https://github.com/dotnet/core-setup/blob/release/1.0.0/build_projects/shared-build-targets-utils/Utils/Monikers.cs#L54-L57

https://github.com/dotnet/core-setup/blob/release/1.0.0/build_projects/shared-build-targets-utils/Utils/Utils.cs#L150-L154

Those methods were copied over from CLI, but aren't invoked, so they can be removed along with this property.